### ISSUE TITLE
BUG: Custom CA certificate throw error on OAuth login.

### ISSUE THREAD

**Issue by depahk** [victorbalssa/abacus#246]
**Describe the bug**
During the initial setup of the app both with oAuth or Personal Access Token, i receive error duting the confirmation.
You can see below the error (italian version, I hope you can understand the error).

**To Reproduce**
With OAuth after the redirect to "abacusfiiiapp://redirect" I receive a translated version of something like "Failed to recover accessToken, Netowk request failed.
With Personal Access Token after the clik on "Log in", "accessToken validation failed, please check your token or backend URL".

**Expected behavior**
Succesfully config :D

**Screenshots**
![photo_2024-02-05_18-24-48](https://github.com/victorbalssa/abacus/assets/44574820/6e7cc573-160e-4c41-98d8-e8ac1ca7df23)
![photo_2024-02-05_18-24-51](https://github.com/victorbalssa/abacus/assets/44574820/7bfa2f88-d148-415f-99e9-88db6b0e993e)

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy A34 5G
 - OS: Android
 - Version 14

**Additional context**
I suspect the the issue come with a custom certificate generated from my internal CA.

**Comment by victorbalssa** [comment]
Hey @depahk 

> I suspect the the issue come with a custom certificate generated from my internal CA.

⬆️  That's the issue, related to [this comment](https://github.com/victorbalssa/abacus/discussions/41#discussioncomment-8138370) I will update the issue title to track my progress on this.

**Comment by depahk** [comment]
Perfect, I didn't see the previous comment :( 
Thanks for the app, it seems really "wow"! I look forward to trying it!

**Comment by paulcalabro** [comment]
@depahk 

Just FYI,I have it working with a custom local root CA. You might find my notes here helpful:

https://github.com/victorbalssa/abacus/issues/256

tl;dr: you should configure your phone to trust the custom local root CA.

**Comment by victorbalssa** [comment]
Done in `0.13.2`.
Let me know if there is still an issue with this.

**Comment by depahk** [comment]
Sorry to be late. Unfortunately @paulcalabro suggestion don't fix my issue.
Also when I opened this issue, I did the test with a local truest CA by my phone and by traefik.

**Comment by victorbalssa** [comment]
Hey @depahk,

Can you retry the same test with the new version `0.13.2` available [here](https://apps.apple.com/us/app/abacus-for-firefly-iii/id1627093491)?

I hope it will fix this issue.

Thanks

**Comment by depahk** [comment]
Tested now: same errors

![photo_2024-03-10_23-39-02](https://github.com/victorbalssa/abacus/assets/44574820/55705d97-286c-435a-8b6b-8e17e7799dab)
![photo_2024-03-10_23-39-00](https://github.com/victorbalssa/abacus/assets/44574820/1f36edfc-f0d6-4bfa-b29c-ae731148ed59)

To be clear: there is network reachability in fact I can use my firefly instance on mobile with Firefox and with Waterfly III app ( C: )

**Comment by victorbalssa** [comment]
I changed only iOS for `0.13.2`, but will check again on Android.
